mslib/mscolab/models.py

```diff
@@ -31,6 +31,21 @@
 from passlib.apps import custom_app_context as pwd_context
 from mslib.mscolab.app import db
 from mslib.mscolab.message_type import MessageType
+import sqlalchemy.types as types
+
+
+class Awaredatetime(types.TypeDecorator):
+    impl = types.DateTime(timezone=True)
```

Comment:
```suggestion
    impl = types.DateTime
```

Removing the timezone=True parameter has the advantage that the underlying database representation stays the same as before in all cases, i.e. no database migration is necessary. This is nicer for backwards-compatibility. The database column then would continue to contain the timestamp without any timezone information. But this is fine, because on the application side we get a guarantee that all timestamps are first converted to UTC and also get returned as UTC. This means the database will only ever contain UTC timestamps.